pkg/snapshot/filesystem.go

```diff
@@ -52,6 +52,14 @@ type ResetFileSystemError struct {
 	Message string
 }
 
+type HostPathFileSystemError struct {
```

Comment:
how about calling it
```suggestion
type HostPathNotFoundError struct {
```

Reply:
sounds good, changed